core/src/main/java/com/novoda/noplayer/PlayerAudioTrack.java

```diff
@@ -2,20 +2,32 @@
 
 public class PlayerAudioTrack {
 
+    private final int groupIndex;
+    private final int trackIndex;
```

Comment:
Both of these fields will be used by `ExoPlayer` because audio files are listed in groups and tracks in a `DASH` manifest. For `MediaPlayer` only the `trackIndex` will be used.